notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>If you pause the warnings, you should note why. For example, Jupiter is a not a star (yet! Unless those pesky aliens from 2010: Odyssey 2 get to it) and therefore will cause issues when we pass it to to the stellar type query</p><p> </p><p>Ideally we would filter out the non-star results; depends on if this is feasible/easy to do.</p><p> </p><p>In either case, we should eliminate the warnings. That makes users panic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>